### PR TITLE
feat: Add filter section for "not-downloaded" button

### PR DESCRIPTION
**Pull request by stoichy**
Screenshot from 2022-10-30 17-25-58

### CODE REVIEW

**Comment by stoichy** [comment]
Issue #1097

File: src/renderer/components/settings-electron.tsx [hunk 1]
```diff
@@ -226,7 +241,7 @@ export const ElectronSettings = observer(
       };
 
       return (
-        <FormGroup label="Include Electron versions:">
+        <FormGroup label="Channels">
```
Comment (aryanshridhar) [inline src/renderer/components/settings-electron.tsx:244]: Let's add back the `:` to keep it consistent with previous UI.

File: src/renderer/components/settings-electron.tsx [hunk 2]
```diff
@@ -199,9 +201,22 @@ export const ElectronSettings = observer(
...
+            label="Not downloaded"
+            onChange={this.handleStateChange}
+          />
+        </FormGroup>
+      );
+    }
 
     /**
-     * Renders the various options for which versions should be displayed
+     * Renders the various options for which versions should be displayedq
```
Comment (aryanshridhar) [inline src/renderer/components/settings-electron.tsx:219]: Typo?

File: src/renderer/components/settings-electron.tsx [hunk 3]
```diff
@@ -199,9 +201,22 @@ export const ElectronSettings = observer(
         </ButtonGroup>
       );
     }
+    private filterSection(): JSX.Element {
+      const { appState } = this.props;
+      return (
+        <FormGroup inline={true} label="Filters:">
```
Comment (aryanshridhar) [inline src/renderer/components/settings-electron.tsx:207]: To keep the design similar to `Channels` formgroup, `inline={true}` can be safely removed.

**Review by aryanshridhar — CHANGES_REQUESTED** [review]
Thanks for working on it 👍 
Left a few minor comments below.

**Comment by stoichy** [comment]
Fixed, please have another look

**Review by codebytere — APPROVED** [review]

**Review by aryanshridhar — APPROVED** [review]

**Pull request merged**